Method onKeyUp in class JsDevReloadHandler cannot be applied to given types

## 🐛 Bug Report

After update the lib to the new version 7.13.0 i got this error when I try to compile to Android:

```
error: method onKeyUp in class ReactGateway cannot be applied to given types;
        return getReactGateway().onKeyUp(this, keyCode) || super.onKeyUp(keyCode, event);
                                ^
  required: int
  found: NavigationActivity,int
  reason: actual and formal argument lists differ in length
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-navigation:compileReactNative62DebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```
## To Reproduce

1. Update lib to latest version 7.13.0
1. Clear android cache `(cd android/ && ./gradlew clean)`
1. Try build app `(react-native run-android)`

## Expected behavior

 I expected build successfully

## Actual Behavior
I get broken build 

## Your Environment

<!-- Include as many relevant details about the environment you experienced the bug in -->

- React Native Navigation version: 7.13.0
- React Native version: 0.64.0
- Platform(s) (iOS, Android, or both?): Android only
- Device info (Simulator/Device? OS version? Debug/Release?): Device | Android OS 10 | Debug and Release

--- a/lib/android/app/src/reactNative51/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative51/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative55/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative55/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative56/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative56/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative57/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative57/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative57_5/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative57_5/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative60/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative60/java/com/reactnativenavigation/react/ReactGateway.java
@@ -56,8 +56,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, final int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

--- a/lib/android/app/src/reactNative62/java/com/reactnativenavigation/react/ReactGateway.java
+++ b/lib/android/app/src/reactNative62/java/com/reactnativenavigation/react/ReactGateway.java
@@ -58,8 +58,8 @@ public class ReactGateway {
 		initializer.onActivityDestroyed(activity);
 	}
 
-	public boolean onKeyUp(final int keyCode) {
-		return jsDevReloadHandler.onKeyUp(keyCode);
+	public boolean onKeyUp(Activity activity, int keyCode) {
+		return jsDevReloadHandler.onKeyUp(activity, keyCode);
 	}
 
     public void onBackPressed() {

